app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -317,6 +325,74 @@ void main() {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/test_fixes'), contains('tech-debt'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/test_fixes/material.expect'),
+            contains('tech-debt'));
+      });
+
+      test('gestures label applied', () {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/gestures'), contains('f: gestures'));
+      });
+
+      test('focus label applied', () {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/focus_node.dart'),
+            contains('f: focus'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/focus_scope.dart'),
+            contains('f: focus'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/focus_manager.dart'),
+            contains('f: focus'));
+      });
+
+      test('routes label applied', () {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/router.dart'),
+            contains('f: routes'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/routes.dart'),
+            contains('f: routes'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/navigator.dart'),
+            contains('f: routes'));
+      });
+
+      test('text input label applied', () {
+        expect(
+            GithubWebhook.getLabelsForFrameworkPath(
+                'dev/integration_tests/web_e2e_tests/test_driver/text_editing_integration.dart'),
+            contains('a: text input'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/text_editing_action.dart'),
+            contains('a: text input'));
+      });
+
+      test('animation label applied', () {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/animation'), contains('a: animation'));
+      });
+
+      test('scrolling label applied', () {
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/widgets/sliver.dart'),
+            contains('f: scrolling'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/material/scrollbar.dart'),
+            contains('f: scrolling'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/flutter/lib/src/rendering/viewport.dart'),
+            contains('f: scrolling'));
+      });
+
+      test('integration_test label applied', () {
+        expect(
+            GithubWebhook.getLabelsForFrameworkPath(
+                'dev/integration_tests/web_e2e_tests/test_driver/text_editing_integration.dart'),
+            contains('integration_test'));
+        expect(GithubWebhook.getLabelsForFrameworkPath('packages/integration_test/lib/common.dart'),
+            contains('integration_test'));
```

Comment:
(this file could be easier to review with some more consistent formatting)

Reply:
I agree, this repo has a formatting script that is enforced though.

Reply:
We could... change that? Making the code hard to review is definitely not the purpose of formatting the code...